migrations/V000003__create_indexes_for_audit_billing_and_fraud_events_tables.sql

```diff
@@ -1,3 +1,3 @@
-CREATE INDEX ON audit.audit_events (event_id);
-CREATE INDEX ON billing.billing_events (session_id);
-CREATE INDEX ON billing.fraud_events (session_id);
+CREATE INDEX audit_events_event_id_idx1 ON audit.audit_events (event_id);
+CREATE INDEX billing_events_session_id_idx1 ON billing.billing_events (session_id);
+CREATE INDEX fraud_events_session_id_idx ON billing.fraud_events (session_id);
```

Comment:
does this need to be `fraud_events_session_id_idx1`?

Reply:
I've commented on the related comment [on #29](https://github.com/alphagov/verify-event-system-database-scripts/pull/29#pullrequestreview-264208578)

Reply:
I guess I was more wondering about why some of them have `1` and why two of them don't?

Reply:
Ah. Yeah, I have no idea. These are names PostgreSQL has generated automatically. 🤷‍♂

Reply:
The idea was to bring the index names inline across all environments, we chose to give them the names that currently existed in production (which, as Alex states, were auto-generated by PostgreSQL).

Given that we did have to manually rename them in other environments to match, then I guess there is the argument that, we should remove the `1` suffix for consistency and rename them in prod also!?